data.sql

```diff
@@ -4,3 +4,12 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Gabumon', date '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Pikachu', date '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)  VALUES ('Devimon', date '2017-05-12', 5, TRUE, 11);
+
+/*Day Two Project*/
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', date '2020-02-08', 0, TRUE, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantom', date '2021-11-15', 2, TRUE, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', date '1993-04-02', 3, FALSE, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', date '2005-06-15', 1, TRUE, 45);
```

Comment:
- [ ] As per the project requirement, Please you are inserting the wrong `weight_kg` values for these data below so make sure you fix  them.
  
  ![image](https://github.com/Kanyorok/vet-clinicdb/assets/92303891/34d6ed58-003f-4ce3-a120-e744760b70c6)